desktop/main.dev.ts

```diff
@@ -242,19 +244,33 @@ const createWindow = async () => {
     shell.openExternal(url);
   });
 
-  const res = await fetch(DISCOVERY_URL);
-  const initialConfig = (await res.json())[0];
   const savedNetId = StoreService.get('netSettings.netId');
-  const cleanStart = savedNetId !== initialConfig.netID;
-  const configFilePath = path.resolve(app.getAppPath(), process.env.NODE_ENV === 'development' ? './' : '../../config', 'config.json');
+  const configFilePath = path.resolve(app.getPath('userData'), 'node-config.json');
+  let netId;
+  let initialConfig;
+  let netConfig;
+  let isDevNet = false;
+  if (process.env.NODE_ENV === 'development' && process.env.DEV_NET_URL) {
```

Comment:
I was hoping that there would be a more straightforward and clean solution without defining `undefined` variables.

Anyway,
```
const isDevNet = process.env.NODE_ENV === 'development' && process.env.DEV_NET_URL;
if (isDevNet) {
```
And there will be no need to switch this flag manually.